core/src/subgraph/instance_manager.rs

```diff
@@ -154,7 +144,7 @@ where
                 deployment.hash.cheap_clone(),
                 manifest,
                 // Allow for infinite retries for subgraph definition files.
-                &self.link_resolver.as_ref().clone().with_retries(),
```

Comment:
Was the `with_retries()` lost in the refactor?

Reply:
https://github.com/graphprotocol/graph-node/pull/3447